app/_landing_pages/gateway/deployment-topologies.yaml

```diff
@@ -8,12 +8,369 @@ rows:
   - header:
       type: h1
       text: "Deployment Topologies"
-  - columns:
+    columns:
       - blocks:
           - type: structured_text
             config:
               blocks:
                 - type: text
                   text: |
-                    @todo
-                    pull content from https://docs.konghq.com/gateway/latest/production/deployment-topologies/, cover what a deployment topology is along with links to more info about each topology
\ No newline at end of file
+                    Kong provides different modes for you to deploy {{site.konnect_short_name}} and {{site.base_gateway}} in. 
+                    Each mode has benefits and limitations, so it is important to consider them carefully when deciding which mode to use to in production installations.
+  - header:
+      type: h2
+      text: "Topologies"
+    columns:
+      - blocks:
+        - type: icon_card
+          config:
+            title: Hybrid mode
+            icon: /assets/icons/deployment.svg
+            cta:
+              url: ./#hybrid-mode
+      - blocks:
+        - type: icon_card
+          config:
+            title: Traditional mode
+            icon: /assets/icons/database.svg
+            cta:
+              url: ./#traditional-mode
+      - blocks:
+        - type: icon_card
+          config:
+            title: DB-less mode
+            icon: /assets/icons/db-less.svg
+            cta:
+              url: ./#db-less-mode
```

Comment:
I don't love these. I think making the icons a bit different could help but overall I don't think it adds much over scrolling down and / or ctrl+f